src/main/java/athena/TaskList.java

```diff
@@ -0,0 +1,136 @@
+package athena;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    public static final String NO_FILTER = "";
+    private ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    private ArrayList<Task> getShortlistedTasks(String instance) {
+        ArrayList<Task> shortlistedTasks = new ArrayList<>();
+        for (Task task: tasks) {
+            if (task.getName().contains(instance)) {
+                shortlistedTasks.add(task);
+            }
+        }
+        return shortlistedTasks;
+    }
+
+    private void printAllTasks() {
+        for (Task task : tasks) {
+            System.out.println(task);
+        }
+    }
+
+    private void printTasksWithFilter(String importanceFilter) {
+        for (Task task : tasks) {
+            if (task.getImportance().equals(importanceFilter)) {
+                System.out.println(task);
+            }
+        }
+    }
+
+
+    private Task createTask(String name, String startTime,
+            String duration, String deadline, String recurrence, String importance, String notes) {
+        Task task = new Task(name, startTime, duration,
+                deadline, recurrence, importance, notes);
+        return task;
+    }
+
+    /**
+     * Returns size of the task list.
+     *
+     * @return Size of the task list
+     */
+    public int getTaskListSize() {
+        return tasks.size();
+    }
+
+    /**
+     * Marks specified task as done.
+     * Lets the user know specified task has been marked as done.
+     *
+     * @param taskNumber Position of task in task list
+     */
+    public void markTaskAsDone(int taskNumber) {
+        tasks.get(taskNumber).setDone();
```

Comment:
Can call the ui method printTaskDone(task) to let users know which task they marked as done